.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/rnaseq') }}
     runs-on: ubuntu-latest
     env:
-      NXF_VER: '20.07.1'
+      NXF_VER: '20.11.0-edge'
```

Comment:
When you change this back to a stable release better, I would think that it's safer to just not specify `NXF_VER` on these tests so that it uses the latest version. Probably more important to test that than the minimum version which doesn't change..?

Reply:
Yup. Makes sense 👍🏽